script_version: 1.0
period: 1
storage_path: "/media/admin/T7"

imu:
  port: "/dev/ttyACM1"
  baudrate: 115200

arduino:
  port: "/dev/ttyACM0"
  baudrate: 9600
  start_bit: 0xFFFF
  timeout: 1